.rubocop.yml

```diff
@@ -1,3 +1,11 @@
+inherit_from: .rubocop_todo.yml
+
+require:
+  - rubocop-performance
+
+AllCops:
+  TargetRubyVersion: 2.7
```

Comment:
This should be 3.0, right?

Reply:
No, we haven't dropped Ruby 2.7 yet.

Reply:
We can use:
```
AllCops:
  TargetRubyVersion: 2.7..3.0
```

Reply:
This is the way IMHO

Reply:
No, you set a minimum version. https://docs.rubocop.org/rubocop/configuration.html#setting-the-target-ruby-version states:
> Users may let RuboCop know the oldest version of Ruby which your project supports with:

Though there's also a line that says it will use the gemspec if it has `required_ruby_version` so perhaps it's redundant and we can drop this line altogether. If that works, I'd prefer that because I dislike duplicating information.